src/app/code/community/Zendesk/Zendesk/etc/config.xml

```diff
@@ -19,18 +19,28 @@
 <config>
     <modules>
         <Zendesk_Zendesk>
-            <version>1.3.1</version>
+            <version>1.4.0</version>
         </Zendesk_Zendesk>
     </modules>
     <zendesk>
-        <provision_url>https://signup.zendesk.com/provisioning/magento/welcome</provision_url>
+        <provision_url>https://support.zendesk.com/hc/en-us/articles/203660046-Magento-Integration-Installing-the-Zendesk-extension-for-Magento</provision_url>
```

Comment:
Why has this URL changed? It should still be pointing to the provisioning service